tools/Makefile.unix

```diff
@@ -459,6 +459,26 @@ endif
 download: $(BIN)
 	$(call DOWNLOAD, $<)
 
+# bootloader
+#
+# Some architectures require the provisioning of a bootloader or other
+# functions required for properly executing the NuttX binary.
+# Make.defs files for those architectures should define a bootloader target
+# with the correct operations for that platform. It will generate an error
+# if the bootloader target is not defined.
+
+bootloader:
+	$(Q) $(MAKE) -C $(ARCH_SRC) bootloader
+
+# clean_bootloader
+#
+# Removes all of the files and directories created by the bootloader target.
+# It will generate an error if the clean_bootloader target is not defined on
+# the architecture-specific Makefile.
+
+clean_bootloader:
```

Comment:
should we add clean_bootloader to the dependence of distclean or clean?

Reply:
For now we can’t because `clean_bootloader` may not be implemented by some archs, which will break `distclean`.
It would require some yet-to-be-defined generic config to decide whether `distclean` should call `clean_bootloader` or not.

Reply:
OK